### ISSUE TITLE
Not getting same FRET hist with and without PIE

### ISSUE THREAD

**Issue by ncodina** [tritemio/FRETBursts#56]
We measured the same sample using one excitation laser and PIE, and after analysis we obtain different FRET histograms. Shouldn't we obtain the same FRET histogram, and then with PIE having the ability to exclude the zero efficiency peak caused by molecules with only donor fluorophore?

<img width="716" alt="screen shot 2017-02-08 at 23 43 07" src="https://cloud.githubusercontent.com/assets/23275480/22762658/675a1590-ee58-11e6-82f5-e743f8ca6468.png">

In both cases we have used an "All photons" search:
d.burst_search(F = 6, m = 10, ph_sel = Ph_sel('all'))
ds = d.select_bursts(select_bursts.size, th1 = 30)

With PIE, we have also tried to select only donor and acceptor photons emitted during donor excitation, to correspond to the photons for non-ALEX data:
d.burst_search(F = 6, m = 10, ph_sel = Ph_sel(Dex=’DAem’))
but we still get the same result.

Do you have some insight into what might be happening?

**Comment by tritemio** [comment]
Just guessing. It can be that your FRET molecules are dimmer or diffuse faster than the D-only. You are using background-dependent burst search (F=6) and the background is 3x higher in the PIE measurement. So in PIE you using an higher threshold that may miss the smaller high-FRET bursts.

Try using a constant-rate burst search in both cases and see if the high-FRET peak is comparable.

Also you need to make sure that the D/A alternation definition of PIE is compatible with what you have with the single laser. If you select a small range in PIE you may loose photons.

**Comment by ncodina** [comment]
But when using "Ph_sel(Dex=’DAem’))" (non-ALEX data), it doesn't use the background of acceptor excitation, right? The background rates for donor excitation are similar to the non-PIE measurement. The high-FRET peak does not appear either.

I selected all photons in the PIE measurement and I tried using: 
d.burst_search(min_rate_cps = 50e3, m = 10, ph_sel = Ph_sel(’all’))
with different rates (5, 10 and 50 kcps) and I can observe a high-FRET population without PIE but not with PIE.
